tests/app/service/test_discovery.py

```diff
@@ -186,7 +186,7 @@ def test_create_ai_apps_model(discovery):
         },
         "framework": {"name": "langchain", "version": "0.1.45"},
         "lastUsed": datetime.datetime(2024, 1, 1, 0, 0, 5),
-        "pebbloServerVersion": "0.1.14",
+        "pebbloServerVersion": "0.1.15",
```

Comment:
Do we need to update this unit test whenever the Pebblo server version is updated?

Reply:
This is failing on my local with the following traceback:
```

E               },
E               'name': 'pytest_app',
E               'owner': 'Shreyas Damle',
E               'pebbloClientVersion': '0.1.1',
E         -     'pebbloServerVersion': '0.1.15',
E         ?                                  ^
E         +     'pebbloServerVersion': '0.1.14',
E         ?                                  ^
E               'pluginVersion': '0.1.1',
E               'retrievals': [],
E           }

test_discovery.py:219: AssertionError

```